libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/BeginSkill.cs

```diff
@@ -180,7 +204,39 @@ protected override string OnComputeId()
                 activity = StringUtils.Ellipsis(Activity?.ToString().Trim(), 30);
             }
 
-            return $"{this.GetType().Name}['{appId}','{activity}']";
+            return $"{GetType().Name}['{appId}','{activity}']";
+        }
+
+        /// <summary>
+        /// Stores the DialogOptions initialized during BeginDialog so they can be restored when the dialog is resumed.
+        /// </summary>
+        private void SaveDialogOptionsToState(DialogInstance instance)
```

Comment:
The comment states 'stores the DialogOptions' and the method is named 'SaveDialogOptionsToState' but nothing is stored or saved, just set in the DialogState.  Can we change this to 'Sets DialogOptions in State', or remove altogether (since it is a one line method).

Reply:
Done